lib/tasks/load_data.rake

```diff
@@ -0,0 +1,36 @@
+require 'csv'
+namespace :load_data do
+  task :csvs => :environment do
+    puts "loading merchants"
+    CSV.foreach('db/data/merchants.csv', :headers => true, :encoding => 'ISO-8859-1:UTF-8') do |row|
+      Merchant.create([
+                      :name => row['name']
+                      ])
+    end
+    puts "loaded #{Merchant.all.count} merchants"
+
+    puts "loading transactions"
+    CSV.foreach('db/data/transactions.csv', :headers => true, :encoding => 'ISO-8859-1:UTF-8') do |row|
```

Comment:
@josh-works that makes total sense. i'm thinking of it from an OO standpoint and single responsibility. even if we aren't loading up all the data unless we're using it, we're separating one big rake task into individual, smaller rake tasks for more flexibility. 

i see what you're saying though. i think the way you have it is perfectly fine - separating it into smaller rake tasks is a small refactor 👍